conf/httpd.conf.d/httpd.graphite

```diff
@@ -101,10 +101,10 @@ $KeepAliveTimeout = "10";
 $KeepAlive = "On";
 $MaxKeepAliveRequests = "100";
 
-$MaxClients = "56";
-$StartServers = "4";
-$MinSpareServers = "4";
-$MaxSpareServers = "20";
+$MaxClients = "2";
+$StartServers = "1";
+$MinSpareServers = "1";
+$MaxSpareServers = "1";
```

Comment:
I would set the max higher than that. Something around 10.
The service is not only used in our dashboard but also by the other servers in active/active setups.

A low minimum is fine OTOH.